include/core/lf_types.h

```diff
@@ -195,15 +196,13 @@ typedef struct event_t event_t;
 
 /** Event activation record to push onto the event queue. */
 struct event_t {
-  instant_t time;     // Time of release.
-  trigger_t* trigger; // Associated trigger, NULL if this is a dummy event.
-  size_t pos;         // Position in the priority queue.
-  lf_token_t* token;  // Pointer to the token wrapping the value.
-  bool is_dummy;      // Flag to indicate whether this event is merely a placeholder or an actual event.
+  pqueue_tag_element_t base; // Elements of pqueue_tag. It contains tag of release and position in the priority queue.
+  trigger_t* trigger;        // Associated trigger, NULL if this is a dummy event.
+  lf_token_t* token;         // Pointer to the token wrapping the value.
+  bool is_dummy;             // Flag to indicate whether this event is merely a placeholder or an actual event.
```

Comment:
Maybe this is_dummy is not needed anymore.  Could just use` trigger == NULL` to signal a dummy event.

Reply:
I removed the flag and used `trigger == NULL` to check whether an event is a dummy event in [739b696](https://github.com/lf-lang/reactor-c/pull/390/commits/739b696d0728fd4cf79ea3b5749c82984d9aaf57).